examples/tonic-key-value-store/build.rs

```diff
@@ -1,3 +1,6 @@
 fn main() {
-    tonic_build::compile_protos("proto/key_value_store.proto").unwrap();
+    tonic_build::configure()
+        .format(false)
```

Comment:
why this?

Reply:
We were getting some CI errors about rustfmt not being installed when publishing the docs. This is an easy fix for that. Don't publish docs for examples regardless.